simpleLooping/incrementBinaryList.py

```diff
@@ -18,3 +18,24 @@
 #   incrementBinaryList([1, 0, 0, 1]) # Return [0, 1, 0, 1]
 #   incrementBinaryList([1, 1, 0, 1]) # Return [0, 0, 1, 1]
 #   incrementBinaryList([1, 1, 1, 1]) # Return [0, 0, 0, 0, 1]
+from typing import List
+
+def incrementBinaryList(lst: List[int]) -> List[int]:
+    lenLst = len(lst)
+    i = 0
+    listBinary = []
+    carry = 1
+
```

Comment:
Why do we have this empty line here? is the initialization above not part of the loop?